packages/Python/lldbsuite/test/lang/swift/swift_interface_shared/libs/Makefile

```diff
@@ -0,0 +1,17 @@
+LEVEL = ../../../make
+
+DYLIB_NAME := $(BASENAME)
+DYLIB_SWIFT_SOURCES := libs/$(BASENAME).swift
+
+# don't use the default swift flags, as we don't want -g for the libraries in
+# this test
+SWIFTFLAGS=-Onone -Xfrontend -serialize-debugging-options -enable-parseable-module-interface
+
+# don't include the wrapped .swiftmodule on linux to make sure we actually use the .swiftinterface
```

Comment:
Add comment: `the issue here is that because the .swiftmodule is being wrapped and linked into the dylib, LLDB will find it there and load it before the .swiftinterface loading path can be exercised.`